### PR TITLE
Some Project Euler problems solved

### PR DESCRIPTION
**Pull request by ashusvnath**

### CODE REVIEW

**Comment by svishwanath-tw** [comment]
Heres a comment on a pull request!

File: problem11.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+       0.upto(self.size - n).map{|k| self[k, n] }
+   end
+end
+
+num_grid = text_grid.split("\n").map{|x| x.split.map(&:to_i)}
+
+each_four = lambda do |x|
+  x.inject([]){|a, v|
+    a + v.slices(4)
+  }.reject{|v|
```
Comment (svishwanath-tw) [inline problem11.rb:35]: Adding a comment on a diff

**Comment by ashusvnath** [comment]
Here is a reply

File: problem13.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+24074486908231174977792365466257246923322810917141
+91430288197103288597806669760892938638285025333403
+34413065578016127815921815005561868836468420090470
+23053081172816430487623791969842487255036638784583
+11487696932154902810424020138335124462181441773470
+63783299490636259666498587618221225225512486764533
+67720186971698544312419572409913959008952310058822
+95548255300263520781532296796249481641953868218774
+76085327132285723110424803456124867697064507995236
+37774242535411291684276865538926205024910326572967
```
Comment (svishwanath-tw) [inline problem13.rb:54]: A comment with spéçîål characters in it.

**Pull request merged**